src/Hive.Core/sia380/sia380.py

```diff
@@ -269,58 +278,131 @@ def main(room_properties, floor_area, T_e, setpoints_ub, setpoints_lb, surface_a
         Q_s[month] = sum(Q_s_per_surface[month])
 
         # Heatgains/-losses ratio (Wärmeeintrag/-verlust Verhältnis), gamma
-        gamma_ub = (Q_i[month] + Q_s[month]) / (Q_T_ub + Q_V_ub)
-        gamma_lb = (Q_i[month] + Q_s[month]) / (Q_T_lb + Q_V_lb)
-        # gamma = (Q_i[month] + Q_s[month]) / (Q_T[month] + Q_V[month])
+        gamma_ub_heating = (Q_i[month] + Q_s[month]) / (Q_T_ub + Q_V_ub_heating)
+        gamma_lb_heating = (Q_i[month] + Q_s[month]) / (Q_T_lb + Q_V_lb_heating)
+        gamma_ub_cooling = (Q_i[month] + Q_s[month]) / (Q_T_ub + Q_V_ub_cooling)
+        gamma_lb_cooling = (Q_i[month] + Q_s[month]) / (Q_T_ub + Q_V_lb_cooling)
+        gamma_ub_heating_no_hr = (Q_i[month] + Q_s[month]) / (Q_T_ub + Q_V_ub_heating_no_heat_recovery)
+        gamma_lb_heating_no_hr = (Q_i[month] + Q_s[month]) / (Q_T_lb + Q_V_lb_heating_no_heat_recovery)
+        gamma_ub_cooling_no_hr = (Q_i[month] + Q_s[month]) / (Q_T_ub + Q_V_ub_cooling_no_heat_recovery)
+        gamma_lb_cooling_no_hr = (Q_i[month] + Q_s[month]) / (Q_T_ub + Q_V_lb_cooling_no_heat_recovery)
 
         # usage of heat gains (Ausnutzungsgrad für Wärmegewinne), eta_g
-        eta_g_lb = calc_eta_g(Q_T_lb, Q_V_lb, gamma_lb, tau)
-        eta_g_ub = calc_eta_g(Q_T_ub, Q_V_ub, gamma_ub, tau)
+        eta_g_lb_heating = calc_eta_g(gamma_lb_heating, tau, False)
```

Comment:
what is the last parameter (bool)? it _looks_ like you changed `calc_eta_g` to have a "function-selection-parameter", which will have a big `if` at the beginning (`if last_parameter:`, ... `else:`...) - why not two functions? That way you can use _names_ to document your code.

Reply:
made `cooling=False` as default argument. I kinda like it better as one function

Reply:
so. i personally like two functions much better. having flag parameters is basically saying: "this is actually two functions in one". but ok. let's do it like this.